core/templates/dev/head/filters.js

```diff
@@ -240,6 +240,15 @@ oppia.filter('parameterizeRuleDescription', [
           replacementText = inputs[varName];
         }
 
+        if (typeof (replacementText) === 'string') {
+          // Replacing all occurances of $ with $$.
+          // When this is used in the next replace
+          // statement, it will be converted back to $.
+          // Also to replace by $$, the appropriate string is $$$
+          // as $$ becomes $
```

Comment:
Add a period "." at the end of the sentence.

Reply:
Done